docs/README.md

```diff
@@ -20,7 +20,7 @@ This document provides an overview of system components, contracts, and user rol
     * [Depositing Into EigenLayer](#depositing-into-eigenlayer)
     * [Delegating to an Operator](#delegating-to-an-operator)
     * [Undelegating or Queueing a Withdrawal](#undelegating-or-queueing-a-withdrawal)
-    * [Completing a Withdrawal as Shares](#completing-a-withdrawal-as-shares)
+    * [Completing a Withdrawal as OwnedShares](#completing-a-withdrawal-as-shares)
```

Comment:
this link is now broken

did you just do a find/replace? can we revert this change? i dont think it applies to slashing stuff.